docs/BUILD

```diff
@@ -136,13 +136,24 @@ pkg_files(
 
 pkg_files(
     name = "sphinx_root",
-    srcs = glob(["root/**/*"]),
+    srcs = glob([
+        "root/**/*.css",
+        "root/**/*.ico",
+        "root/**/*.js",
+        "root/**/*.json",
+        "root/**/*.pb",
+        "root/**/*.png",
+        "root/**/*.proto",
+        "root/**/*.rst",
+        "root/**/*.svg",
+        "root/**/*.yaml",
```

Comment:
Do we expect folks will miss this when adding other file types to the docs set? Should it be documented anywhere or is docs/BUILD self-explanatory?